app/code/Magento/Catalog/Model/Product/Type/AbstractType.php

```diff
@@ -9,9 +9,11 @@
 use Magento\Catalog\Api\ProductRepositoryInterface;
 use Magento\Framework\App\Filesystem\DirectoryList;
 use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Filesystem\DriverInterface;
 
 /**
- * @api
```

Comment:
Please, do not remove the `@api` directive. It's an important marker for backward compatibility checks.